design/components/conversion-block.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import FoursevensIcon from '../assets/foursevens-icon.svg';
+import { styled } from './stitches.config';
+
+const ContentArea = styled('div', {
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  alignSelf: 'center',
+  height: 140,
+});
+
+const ContentAreraWrapper = styled('div', {
+  backgroundColor: '$primary2',
+  overflow: 'hidden',
+  position: 'relative',
+  padding: '0 50px 0 50px',
+
+  '> svg': {
+    width: 200,
+    position: 'absolute',
+    top: -55,
```

Comment:
The combination of `width` and `top` is strange. We should not have to set such a magic top value.

Reply:
Unset `width` and set `top: 20px` seems to work good enough.